esp/mlrs-wireless-bridge/mlrs-wireless-bridge.ino

```diff
@@ -102,25 +102,30 @@ Troubleshooting:
 //**********************//
 //*** WiFi settings ***//
 
+#define USE_BROADCAST_FOR_UDP // Comment out if you want to use only the first WiFi client to connect
+
 // for TCP, UDP (only for these two)
 // ssid = "" results in a default name, like "mLRS-13427 AP UDP"
 // password = "" makes it an open AP
 String ssid = ""; // "mLRS AP"; // Wifi name
 String password = ""; // "thisisgreat"; // WiFi password
 
+// Our IP address in AP mode
 IPAddress ip(192, 168, 4, 55); // connect to this IP // MissionPlanner default is 127.0.0.1, so enter 192.168.4.55 in MP
 
 int port_tcp = 5760; // connect to this port per TCP // MissionPlanner default is 5760
 int port_udp = 14550; // connect to this port per UDP // MissionPlanner default is 14550
 
-// for UDPCl (only for UDPCl)
+// for UDPCl (only for UDPCl) Station mode
 // network_ssid = "" results in
 // - a default name, like "mLRS-13427 STA UDPCl"
 // - and a default password which includes your mLRS bindphrase, like "mLRS-mlrs.0"
 String network_ssid = ""; // name of your WiFi network
 String network_password = "****"; // password to access your WiFi network
 
-IPAddress ip_udpcl(192, 168, 0, 164); // connect to this IP // MissionPlanner default is 192.168.0.164
+// Uncomment and edit the next two lines only if your AP has no DHCP server
```

Comment:
maybe `// Uncomment this line if your access point/hot spot has no DHCP server (not very likely)`